library/asn1parse.c

```diff
@@ -219,11 +244,23 @@ int mbedtls_asn1_get_bitstring( unsigned char **p, const unsigned char *end,
         return( MBEDTLS_ERR_ASN1_OUT_OF_DATA );
     bs->len -= 1;
 
-    /* Get number of unused bits, ensure unused bits <= 7 */
-    bs->unused_bits = **p;
-    if( bs->unused_bits > 7 )
-        return( MBEDTLS_ERR_ASN1_INVALID_LENGTH );
-    (*p)++;
+    /* Get number of unused bits */
+    if ( ( bs->unused_bits = *(*p)++ ) )
+    {
+        unsigned char last;
+
+	/*
```

Comment:
Please don't use tab characters - this is enforced by the CI, so the PR won't pass when any tabs are used for indenting